### ISSUE TITLE
[Bug]: LNURL-Channel doesn't work properly 

### ISSUE THREAD

**Issue by limpbrains** [synonymdev/bitkit-react-native#1490]
### Describe the bug

After scanning lnurl-channel it usually needs a few app restarts to finally open a channel

Also on channel details this new channel has no name before it is opened 

### Reproduce

1. Scan LNURL-channel
3. Watch the app

### Screenshots / Recording

https://github.com/synonymdev/bitkit/assets/155891/1473abdf-e83b-4bad-a01f-bc9bc2e24cc6

### Operating system

iOS

### Bitkit version

v94

### Log output
[bitkit_ldk_logs_1705059027826.zip](https://github.com/synonymdev/bitkit/files/13918516/bitkit_ldk_logs_1705059027826.zip)

**Comment by JeanlChristophe** [comment]
From Bitkit I tried to open a channel with the Blocktank widget.

When I was about to pay, I scanned the LNURL invoice.

Bitkit: I received the error message "Time out error". The amount had gone through. However, there is no trace in the Bitkit transaction history.

Blocktank widget: Transaction successful and asks me to claim the channel.

In order to claim the channel, I scan the QR code, I get the error message: "Unable to Open Channel (LNURL)"

ERROR 1 - no history of the transaction
ERROR 2 - impossible to open channel
![Screenshot_20240117-174255](https://github.com/synonymdev/bitkit/assets/46056211/16396f50-9cdb-4979-9bbf-8cfbf324de29)
![Screenshot_20240117-174537](https://github.com/synonymdev/bitkit/assets/46056211/73fe477f-c42a-4d3f-8894-17cae6c6ccf7)
